package.json

```diff
@@ -1,15 +1,14 @@
 {
   "name": "ep_signatureauth",
-  "version": "1.1.3",
+  "version": "1.2.0",
   "description": "Enable secure sign-in and signature verification from Canvas LMS with Etherpad collaborations tool.",
   "author": "emdurr <ericdurr@atomicjolt.com> (Eric Durr)",
   "contributors": [],
   "dependencies": {
-    "async": "^1.2.1",
+    "async": "^1.5.1",
     "async-stacktrace": "^0.0.2",
     "cookies": "^0.7.0",
-    "js-yaml": "^3.8.2",
-    "timing-safe-equal": "^1.0.0"
+    "js-yaml": "^3.13.0"
   },
   "engines": {
     "node": ">= 0.6.0"
```

Comment:
Seems like the new version of etherpad requires a newer version of node. Would it make sense to bump this too?

Reply:
Yep, I made it 10